src/classes/PDF.ts

```diff
@@ -4,11 +4,49 @@ import OneBlinkAPI from '../lib/one-blink-api'
 import { ConstructorOptions } from '../types'
 
 export default class PDF extends OneBlinkAPI {
+  /**
+   * #### Example
+   *
+   * ```typescript
+   * const OneBlink = require('@oneblink/sdk')
+   *
+   * const options = {
+   *   accessKey: '123455678901ABCDEFGHIJKL',
+   *   secretKey: '123455678901ABCDEFGHIJKL123455678901ABCDEFGHIJKL',
+   * }
+   * const keys = new OneBlink.Keys(options)
+   * ```
+   */
   constructor(options: ConstructorOptions) {
     options = options || {}
     super(options.accessKey, options.secretKey)
   }
 
+  /**
+   * ```javascript
+   * const fs = require('fs')
+   * const util = require('util')
+   * 
+   * const writeFileAsync = util.promisify(fs.writeFile)
+   * 
+   * async function run() {
+   *     const buffer = await pdf.generateFormSubmissionPDF({
+   *     formId: 1,
+   *     submissionId: 'c63ec3ac-12ab-447c-951c-2815d0e6fc24',
+   *     isDraft: false,
+   *     includeSubmissionIdInPdf: false,
+   *     excludedElementIds: ['1ae6d5f5-eade-411c-b85a-45fe40fe469e'],
+   *   })
+   *   await writeFileAsync('./submission.pdf', buffer, 'binary')
+   * }
+   * ```
+   * @param options.formId The exact identifer of the form you wish to generate a pdf for 
+   * @param options.submissionId The submission identifier generated after a successful form submission.
+   * @param options.isDraft `true` if the submission is a draft submission, otherwise `false` 
+   * @param options.includeSubmissionIdInPdf `true` to include the submission identifier in the PDF, otherwise `false` 
+   * @param options.excludedElementIds Array of elements ids to be excluded from the PDF document  
+   * @param options.usePagesAsBreaks Whether pages in the form submission should translate to page breaks in the PDF
+   */
```

Comment:
These descriptions should be below on the actual type. Check out the `generateFormUrl` function in the `Forms.ts` file to get an idea if how it should look.